src/main/java/com/google/sps/data/AvailabilityTimeSlot.java

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.auto.value.AutoValue;
+
+/**
+ * An AvailabilityTimeSlot represents a table entry on the availability page that has a date and
+ * time (utcEncoding), a visual represenation of the time (timeString), and a class that is set
+ * based on whether or not the slot has already been selected in a previous visit to the page (green
+ * = selected = 'table-success').
```

Comment:
Including details about the JavaScript implementation should not really be necessary and makes the details of this class look "leaky".

Reply:
Done